### DISCUSSION TITLE
Display the grid

### DISCUSSION THREAD

**Discussion by ClearlyAwesome** [google/s2geometry#328] [Q&A]
Is there a way to display the S2 grid as an overlay even when the map moves?

**Comment by smcallis** [comment]
What's the context here?  S2 doesn't draw graphics by itself.

**Reply by ClearlyAwesome** [reply]
For sure. I understand that part but I know s2 divides the earth into squares of different 'levels'. I want to display those squares as an overlay. Would I just use the native mkMapOverlay in order to display said squares?

**Reply by smcallis** [reply]
It sounds like mkMapOverlay is an Apple API?  We don't know much about that.  You can always get the vertices of cells and draw them like polygons if needed.
